Multisig/UI/Settings/OwnerKeyManagement/AddKeyAsNewOwner/ReviewReplaceOwnerTxViewController.swift

```diff
@@ -0,0 +1,94 @@
+//
+// Created by Dirk Jäckel on 03.05.22.
+// Copyright (c) 2022 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class ReviewReplaceOwnerTxViewController: ReviewSafeTransactionViewController {
```

Comment:
I thought this class should be a subclass from ReviewSafeChangeTransactionViewController @DmitryBespalov ?

Reply:
I would rather rename ReviewChangeSafeTxViewController to ReviewAddOwnerTxViewController. And not have another layer that is not necessary. 

What would be in ReviewChangeSafeTxViewController as super class to ReviewAddOwnerTxViewController and ReviewReplaceTxViewController?

Reply:
at least the 'add owner' parameters would be different than 'replace owner' parameters, and also 'remove owner' parameters. I thought about a separate base class for all review controllers for changing settings

Reply:
and the cells would be different based on the use case

Reply:
I still don't get it. Why not use `ReviewSafeTransactionViewController` as the common base class for all owner modifications (add/replace/delete)? Maybe even all safe changes (enable Module/disable Module/change threshold)?  Why do we need `ReviewChangeSafeTxViewController` between `ReviewSafeTransactionViewController` and `ReviewReplaceOwnerTxViewController`? What would be in `ReviewChangeSafeTxViewController`?

Reply:
Oh I see - you are saying that the change safe transactions are the same as other - send funds transfer, or possibly custom contract interaction transaction and we can have only one base class to inherit from. I was thinking that they would be different and so proposed to have a deeper class hierarchy

Reply:
Dima's point is to make ReviewAddOwnerTxViewController and ReviewReplaceOwnerTxViewController as subclasses from ReviewChangeSafeTxViewController, as they both has common parts, we agreed that we will do it when working on ReviewReplaceOwnerTxViewController,

Reply:
What would be in this class? IMHO at the moment it would be empty. If we really need it later, we can introduce it.